systemd/cloudlinux-nsqd.service

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=Imunify360 cloudlinux nsqd
+Documentation=https://docs.imunify360.com/
+PartOf=imunify360.service
+
+[Service]
+CPUAccounting=true
+MemoryAccounting=true
+BlockIOAccounting=true
+Slice=Imunify-cloudlinux_nsqd.slice
+Environment=GOGC=10
+ExecStart=/usr/sbin/cloudlinux-nsqd --data-path /var/imunify360/ --use-unix-sockets --tcp-address /var/run/nsqd.sock --http-address /var/run/nsqd-http.sock
```

Comment:
I suggest to use data-path is `/var/imunify360/nsqd/`

Reply:
/var/imunify360/nsqd/ is not created at a time inside defence360 that's why running nsqd on top of that folder will cause nsqd fail with no such directory. I am not sure it make sense to mkdir -p /var/imunify360/nsqd on ExecStartPre but sqlite files exists inside that /var/imunify360 as well as other folders of subservices.

Reply:
`/var/imunify360/` a lot of directories right now and nsqd has a lot of files

Reply:
Should we specify /var/imunify360/nsqd and then create it inside defence360 repo?

Reply:
```
then create it inside defence360 repo
```
definitely no.